docs/troubleshooting.md

```diff
@@ -0,0 +1,130 @@
+# Troubleshooting Marketplace Operator
+
+This document lists out some common failures related to the Marketplace operator a user might encounter, along with ways to troubleshoot the failures. If you encouter a failure related to the Marketplace operator that is not listed in this document, but should be, please open an issue to have the failure appended to this document. 
+
+
+Table of contents
+===================
+
+1. [No packages show up in the UI](https://github.com/anik120/operator-marketplace/blob/troubleshooting-guide/docs/troubleshooting.md#no-packages-show-up-in-the-ui)
```

Comment:
The links here are from my forked repository. Does anyone have any suggestion on how to include links here before the PR goes through?